youtube_dl/downloader/external.py

```diff
@@ -47,12 +47,20 @@ def supports(cls, info_dict):
 
     def _option(self, command_option, param):
         param = self.params.get(param)
-        if param is None:
+        if param is None or param is False:
```

Comment:
It may be useful if default option's value is `None` (that I overlooked for `nocheckcertificate`), but I guess it's better to remove it and require usage of argless option method in such cases.